src/styles/components/tile.scss

```diff
@@ -1,82 +1,83 @@
 @import '../settings';
 @import '../colours';
 
-$hover-lightness-value: -15%;
+$animation-time: 0.25s;
 
-$size: 15rem;
+$tile: '.tile';
 
-.tile {
+#{$tile} {
   position: relative;
-  background-color: $colour-sea-blue;
+  background: var(--tile-background, #{$colour-sea-blue});
+  &.tile-gradient {
+    background: linear-gradient(rgba(255, 255, 255, 0.5), transparent)
+      var(--tile-background, #{$colour-sea-blue});
+  }
   display: inline-block;
-  margin: $global-margin / 2;
   text-align: center;
   color: $colour-sky-white;
-  h1,
-  h2,
-  h3,
-  h4 {
+  &__header,
+  &__subtitle {
+    // hack for rendering before svg
+    opacity: 0.9;
     color: $colour-sky-white;
+    text-shadow: 0 4px 4px rgba(0, 0, 0, 0.4);
+    margin: 0;
   }
-  width: $size;
-  height: $size;
-  padding: $global-padding 0;
+  padding: $global-padding/2 0 0 0;
   cursor: pointer;
-  transition: background-color 0.25s ease-out, color 0.25s ease-out;
-  &:hover {
-    background-color: scale-color(
-      $colour-sea-blue,
-      $lightness: $hover-lightness-value
-    );
-    color: $colour-sky-white;
-  }
+
   &__header {
     display: block;
   }
-  &__content {
-    padding: 0.7em;
+  &__description {
+    transition: height $animation-time linear;
```

Comment:
Don't transition height, it's expensive to transition (https://csstriggers.com/height). Transition transform instead (https://csstriggers.com/transform)